### DISCUSSION TITLE
Multiple row select with checkbox and delete selected row

### DISCUSSION THREAD

**Discussion by kitodev** [vmware/clarity#6323] [Q&A]
First of all, I'm tried to made multiple select row with checkbox like this and some buttons, delete checked row:
![64577932-76a8f580-d3b0-11e9-8bef-3744b0f654d1](https://user-images.githubusercontent.com/13621062/132066914-9a5747be-1b71-4710-89bb-e12249148033.png)
I dont know it is possible to do, or no? I didnt find any solution in clarity documentation, for do this like in picture. I use  clarity in Angular v12.2.0

**Comment by bbogdanov** [comment]
Hi @kitodev,

[Here](https://angular.clarity.design/documentation/datagrid/batch-action) you can see an example of what you are trying to achieve

**Reply by kitodev** [reply]
Hi @bbogdanov,

Thank for your reply, but there is not enough information and code examples, how can I delete selected items.

**Reply by bbogdanov** [reply]
We do not provide a `delete` example cause it is simple deleting elements from the array passed in `ngFor` or `clrDgItems`. How you are going to do that we left it for you to decide. 

[Here](https://github.com/vmware/clarity/blob/angular/projects/website/src/app/documentation/demos/datagrid/batch-action/batch-action.ts#L49) you can see some code related to `delete` rows.
